.pre-commit-config.yaml

```
@@ -1,7 +1,7 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 exclude: "CHANGELOG.md|.copier-answers.yml|.all-contributorsrc"
-default_stages: [commit]
+default_stages: [Nonepre-commitNone]
```

```suggestion
default_stages: [pre-commit]
```
(https://github.com/pre-commit/pre-commit/issues/2732)